src/tiles/list/tile-list-compact-header.ts

```diff
@@ -0,0 +1,75 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { SortParam } from '@internetarchive/search-service';
+import eyeIcon from '../../assets/img/icons/eye';
+import { TileModel } from '../../models';
+
+@customElement('tile-list-compact-header')
+export class TileListCompactHeader extends LitElement {
+  @property({ type: Object }) model?: TileModel;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Object }) sortParam?: SortParam;
+
+  render() {
+    return html`
+      <div id="list-line-header" class="${this.classSize}">
+        <div id="thumb"></div>
+        <div id="title">Title</div>
+        <div id="date">Date Archived</div>
+        <div id="creator">Creator</div>
+        <div id="views">${eyeIcon}</div>
+        <div id="icon"></div>
+      </div>
+    `;
+  }
+
+  private get classSize(): string {
+    return (this.currentWidth ?? 531) < 530 ? 'mobile' : 'desktop';
```

Comment:
Can you pass the [mobileBreakpoint](https://github.com/internetarchive/iaux-collection-browser/pull/10/files#diff-3f10ef47f877ef23eaab33c8a524ccd412dd0aa6ee7b067e94b28a2578943434R107) down into here instead of hardcoding the numbers?